model/DataBaseAccess.php

```diff
@@ -360,6 +361,7 @@ public function createTables()
         $table->addColumn(self::COLUMN_RESOURCE_ID, 'string', ['notnull' => null, 'length' => 255]);
         $table->addColumn(self::COLUMN_PRIVILEGE, 'string', ['notnull' => null, 'length' => 255]);
         $table->setPrimaryKey([self::COLUMN_USER_ID, self::COLUMN_RESOURCE_ID, self::COLUMN_PRIVILEGE]);
+        $table->addIndex([self::COLUMN_RESOURCE_ID], self::INDEX_RESOURCE_ID);
```

Comment:
Do we have a DBMS, supporting reverse indexes?
I think using one may improve the performance even more, as the significant data is usually at the end of the `resource_id` value.

Reply:
postgres doesn't support reverse indexes, but we can do a trick by adding an extra field with reversed value or using functional indexes. 
but I don't think it is noticeably faster in our case than b-tree. 
but anyway it would be interesting to check :)

Reply:
Shall we raise a technical task not to forget about it?